src/main/scala/com/raphtory/core/components/querytracker/QueryProgressTracker.scala

```diff
@@ -62,14 +53,11 @@ import scala.collection.mutable.ListBuffer
   * import com.raphtory.GraphState
   * import com.raphtory.output.FileOutputFormat
   *
-  * val customConfig = Map(("raphtory.pulsar.endpoint", "localhost:1234"))
-  * Raphtory.createClient("deployment123", customConfig)
   * val graph = Raphtory.createGraph(ResourceSpout("resource"), LOTRGraphBuilder())
```

Comment:
Make sure this is updated with the latest changes (`createGraph` doesn't exist currently). I know this is a bit of a pain given how frequently things have been changing.